### DISCUSSION TITLE
Some problems of tinc in Android

### DISCUSSION THREAD

**Discussion by jenarlion** [pacien/tincapp#111] [Q&A]
Guys, excuse me

After I installed Android version 0.33 software and executed the command to generate networks and nodes, the corresponding folders and configuration files did not appear in the mobile phone directory.

I'm here to ask you. Please help me answer it. Thank you

**Comment by pacien** [comment]
Hello,

The configuration is currently accessible and editable through an embedded FTP server, in order to work around some limitations in recent versions of Android, until I figure out a simpler way to manage permissions.

The following section of the documentation covers that aspect and explains how to access the generated configuration files:
https://tincapp.pacien.org/doc.html#configuration-access

**Reply by borygo77** [reply]
Hi Pacien! I broke my ftp going through setting files and changing ftp user and pass.
Currently on Android 10 with root.
What's the path to files without ftp?
Need to save my keys and delete app to reinstall fresh. Thanks for your hard work! 😊

**Reply by borygo77** [reply]
> Hi Pacien! I broke my ftp going through setting files and changing ftp user and pass.
> Currently on Android 10 with root.
> What's the path to files without ftp?
> Need to save my keys and delete app to reinstall fresh. Thanks for your hard work! 😊 

Right. Fixed this with root enabled file explorer. Paths are in app 😁 Forgot about it 😉

**Comment by jenarlion** [comment]
OK，I see
I will try it later，thank you



---Original---
From: ***@***.***&gt;
Date: Mon, Apr 11, 2022 07:43 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [pacien/tincapp] Some problems of tinc in Android (Discussion#111)




 
Hello,
 
The configuration is currently accessible and editable through an embedded FTP server, in order to work around some limitations in recent versions of Android, until I figure out a simpler way to manage permissions.
 
The following section of the documentation covers that aspect and explains how to access the generated configuration files:
 https://tincapp.pacien.org/doc.html#configuration-access

**Comment by kb3ien** [comment]
I understand why this was done, but for the love of all that is holy, just provide a hyperlink in the app and in plain sight. This thread took too long to find !
